Proofreading MO Guide (#11605)
openvinotoolkit/openvino: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_GPT2.md

# Convert ONNX GPT-2 Model {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_GPT2}

> [Public pre-trained GPT-2 model](https://github.com/onnx/models/tree/master/text/machine_comprehension/gpt-2) is a large

Author: ```suggestion
[Public pretrained GPT-2 model](https://github.com/onnx/models/tree/master/text/machine_comprehension/gpt-2) is a large
```

transformer-based language model with a simple objective: predict the next word, given all of the previous words within some text.

## Download the Pre-Trained Base GPT-2 Model

To download the model, go to [this model](https://github.com/onnx/models/blob/master/text/machine_comprehension/gpt-2/model/gpt2-10.onnx), and press **Download**.

To download the model and sample test data, go to [this model](https://github.com/onnx/models/blob/master/text/machine_comprehension/gpt-2/model/gpt2-10.tar.gz), and press **Download**.

> ## Convert ONNX* GPT-2 Model to IR

Author: ```suggestion
## Convert ONNX GPT-2 Model to IR
```

> Generate the Intermediate Representation (IR) of the model GPT-2 by running the Model Optimizer with the following parameters:

Reviewer: ```suggestion
Generate the Intermediate Representation of the model GPT-2 by running Model Optimizer with the following parameters:
```
